exporter/otlpexporter/otlp.go

```diff
@@ -195,7 +196,7 @@ func (oce *otlpExporter) pushMetricsData(ctx context.Context, md consumerdata.Me
 
 	// Perform the request.
 	request := &otlpmetrics.ExportMetricsServiceRequest{
-		// TODO: translate from md to ResourceMetrics field
+		ResourceMetrics: metricstranslator.OCToOTLP(md),
```

Comment:
Please also enable metrics in lines 93-94.

Reply:
done